hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml

```diff
@@ -189,6 +189,67 @@
+                                        width="40" height="40"
+                                        class="oe_kanban_avatar float-left mr4"/>
+                                </div>
+                                <div class="col-10">
+                                    <strong class="o_kanban_record_title"><t t-esc="record.employee_id.value"/></strong>
+                                    <div class="text-muted o_kanban_record_subtitle">
+                                        <t t-esc="record.complete_name.value"/>
+                                    </div>
+                                    <div class="o_dropdown_kanban dropdown" groups="base.group_user">
+                                        <a role="button" class="dropdown-toggle o-no-caret btn" data-toggle="dropdown" href="#" aria-label="Dropdown menu" title="Dropdown menu">
+                                            <span class="fa fa-ellipsis-v"/>
+                                        </a>
+                                        <div class="dropdown-menu" role="menu">
+                                            <a t-if="widget.editable" role="menuitem" type="edit" class="dropdown-item">Edit</a>
+                                            <a t-if="widget.deletable" role="menuitem" type="delete" class="dropdown-item">Delete</a>
+                                            <a t-if="record.state.raw_value == 'draft'" name="action_timesheet_confirm" type="object"
+                                                role="menuitem" class="dropdown-item">Submit to Reviewer</a>
+                                            <a t-if="record.state.raw_value == 'confirm' and record.can_review.value" name="action_timesheet_done" type="object"
+                                                role="menuitem" class="dropdown-item">Approve</a>
+                                            <a t-if="record.state.raw_value == 'confirm' and record.can_review.value" name="action_timesheet_refuse" type="object"
+                                                role="menuitem" class="dropdown-item">Refuse</a>
+                                            <a t-if="record.state.raw_value == 'done' and record.can_review.value" name="action_timesheet_draft" type="object"
+                                                role="menuitem" class="dropdown-item">Set to Draft</a>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="row no-gutters justify-content-end">
+                                <div class="col-3"><span class="text-muted">Period</span></div>
+                                <div class="col-7"><t t-esc="record.date_start.value"/> - <t t-esc="record.date_end.value"/></div>
+                                <div class="w-100"></div>
+                                <div class="col-3"><span class="text-muted">Total time</span></div>
```

Comment:
```suggestion
                                <div class="col-3"><span class="text-muted text-nowrap">Total time</span></div>
```
I put this before, but you may have not seen it due to the github indentation. Sorry.

Reply:
Thanks! Done for both lines